Build/linux/build.sh

```diff
@@ -42,35 +38,28 @@ function release {
 	mkdir -p ../../Bin/Release
 	cd release
 	PATH=$PATH:/usr/local/bin/
-	cmake ../../../Source 					\
+	cmake ../../.. 					\
 		-DCMAKE_BUILD_TYPE=Release			\
 		-DCMAKE_C_COMPILER=$CMAKE_COMPILER		\
-		-DCMAKE_CXX_COMPILER=$GXX_COMPILER		\
 		-DCMAKE_ASM_NASM_COMPILER=$CMAKE_ASSEMBLER	\
-		-DCOMPILE_AS_CPP=$COMPILE_AS_CPP		\
 		-DCMAKE_AR=`which $AR_COMPILER`			\
 		-DCMAKE_RANLIB=`which $RANLIB_COMPILER`	\
-		
-
-	python ../scrub-make.py
 	
 	# Compile the Library	
-	make HevcEncoder -j
-	cp Lib/System/libHevcEncoder.so ../../../Bin/Release/libHevcEncoder.so
+	make -j SvtHevcEnc
 	# Compile the App
-	make HevcEncoderApp
-	cp App/HevcEncoderApp ../../../Bin/Release/HevcEncoderApp
-	cd ..	
+	make -j SvtHevcEncApp
+	# Compile the Simple App
+	# make -j SvtHevcEncSimpleApp
+	cd ..
 }
 
 # Defines
 CMAKE_ASSEMBLER=yasm
 GCC_COMPILER=gcc
 ICC_COMPILER=/opt/intel/composerxe/bin/icc
-GXX_COMPILER=g++
```

Comment:
Have we ever tested ICC_COMPILER? Do we need to keep it?

Reply:
we can remove IIC for now